Typecheck fails for Select.ts

# Bug Report

I've imported
```
import { Props } from 'payload/components/fields/Select'
``` 
in my project. Now when I run a typecheck script (`tsc -b`) in my project, the lib files get checked aswell and I get a type error.

## Current Behavior

Typecheck fails with:
```
node_modules/payload/components/fields/Select.ts:2:17 - error TS2305: Module '"../../dist/admin/components/forms/field-types/Select/types"' has no exported member 'Option'.
```

## Expected Behavior
Typecheck is successful without the need to exclude the libs from type checking.

## Steps to Reproduce

1. Create a new Payload Project
2. import `Props` from `payload/components/fields/Select`
3. run a typecheck (`"typecheck": "tsc -b"` in package.json)

## Detailed Description

Payload Version 1.1.26

--- a/components/fields/Select.ts
+++ b/components/fields/Select.ts
@@ -1,2 +1,2 @@
 // eslint-disable-next-line import/named
-export { Props, Option } from '../../dist/admin/components/forms/field-types/Select/types';
+export { Props } from '../../dist/admin/components/forms/field-types/Select/types';

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -22,7 +22,10 @@
       ],
       "payload/types": [
         "./src/types/index.ts"
-      ]
+      ],
+      "payload/types": [
+        "./src/fields/config/index.ts"
+      ],
     },
     "esModuleInterop": true, /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     "resolveJsonModule": true,

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -21,10 +21,7 @@
         "./src/auth/types.ts"
       ],
       "payload/types": [
-        "./src/types/index.ts"
-      ],
-      "payload/types": [
-        "./src/fields/config/index.ts"
+        "./src/types/index.ts",
       ],
     },
     "esModuleInterop": true, /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */

